academy/ibc/ibc-token-lifecycle.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "IBC Token Lifecyle"
+order: 99
+description: Inter-chain token transfers
```

Comment:
I would recommend either saying "IBC token transfers" or "cross-chain token transfers". Or is this the terminology you found in the documentation?